Source/JavaScriptCore/b3/B3ReduceStrength.cpp

```diff
@@ -3652,9 +3654,12 @@ class ReduceStrength {
 
 bool reduceStrength(Procedure& proc)
 {
+    WTFBeginSignpostAlways(nullptr, JSCJITCompiler);
```

Comment:
Let's put behind `Options::useCompilerSignpost()`.

Reply:
Whoops, didn't mean to upload that... Will remove.